docs/src/components/FooterDark.js

```diff
@@ -41,6 +41,13 @@ export default function FooterDark(props) {
         <a className="text-xs pt-2" href="/overview" rel="noopener noreferrer">
           Documentation
         </a>
+        <a
+          className="text-xs pt-1"
+          href="https://medium.com/hyperjump-tech"
+          rel="noopener noreferrer"
+        >
+          Hyperjump's Medium
```

Comment:
This text should be "Blog". Easier to understand that way.

Reply:
Agree, "Hyperjump" is redundant here. Either medium articles or something more specific.